### ISSUE TITLE
Keyboardsplitter sometames makes keyboards stop working 

### ISSUE THREAD

**Issue by CockyRichard** [djlastnight/KeyboardSplitterXbox#133]
Hello, first off i want to thank you for making this program. Im a tournament organizer for a small local fighting game scene and this program is our only way to connect two KB players in a setup without issues. 

Now here is the problem. 

Sometimes when we connect the keyboard or swap keyboards so other players may use their own keyboard on the setup that we use the PC just straight up does not read key presses anymore from no other keyboards. 

This has happened on both a laptop that we use (its a high end one, not a specs problem) and a full rig on a gaming bar.

The only solution we found is restarting the whole PC, but this is making us the event organizers lose our marbles because after a restart there is a ton of things we need to set up in the PCs for the events, and a full restart is really time consuming.

If you need more information on the problem i will gladly get anything you need.

**Comment by MaguirePool** [comment]
did u put the BLOCK KEYBOARD option?? if yes then it blocks the keyboard

**Comment by CockyRichard** [comment]
> did u put the BLOCK KEYBOARD option?? if yes then it blocks the keyboard

Yeah, but its not that it blocks keyboards on use, thats expected, its that after pluging out keyboards and putting others in it stops detecting all keyboards, like they were never plugged.

**Comment by djlastnight** [comment]
Interesting. I did test hot swapping keybords/mice at the past (Windows 7) and it used to work. Bear in mind KS is using third party keyboard driver, which recently has many issues https://github.com/oblitum/Interception/issues (I do not know what Microsoft did to W10, but many ppl complains about keyboard/mouse issues, related to this driver). Probably there is modern way for input interception, but someone needs to seach for or even implement an API, so we can update KS

**Comment by CockyRichard** [comment]
I wasnt expecting an ansewer but thanks for the info! 

Nowdays most of our KB players have swapped to Hitboxes and arcade sticks, but im still interested in seeing how this develops because with Riot's Project L game releasing soon~ish i know there is gonna be an influx of new Fighting game Players that wanna use keyboards.

I hope in the future there is a better way of doing all this multi-keyboard thing better. Damn you Microsoft and you single keyboard pc shenanigans
